src/components/environment-variables/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import usePollEnvVars from './use-poll-env-vars';
+import EnvironmentVariablesList from './environment-variables-list';
+
+const EnvironmentVariables = (props) => {
+  const {
+    appName,
+    envName,
+    componentName,
+    componentType,
+    includeRadixVars,
+    readonly,
+  } = props;
+  const [context, setContext] = useState({ paused: false });
```

Comment:
React has a concept named context (https://reactjs.org/docs/context.html) which has a different usage. Perhaps context and useContext should be renamed to it's actual usage; enabling and disabling envvar polling=> const [pollingEnabled, setPollingEnabled] and a bool instead of an object.

Reply:
renamed to `poolingState` and `setPoolingState`